packages/blade/src/components/Amount/Amount.stories.tsx

```diff
@@ -37,6 +38,22 @@ const Page = (): React.ReactElement => {
         export default App;
         `}
       </Sandbox>
+      <Box marginBottom="spacing.10">
+        <Text>
+          The Amount component automatically formats numbers based on the user's browser locale
```

Comment:
you can use the propname `propsDescription` on the StoryPageWrapper component and put this information there you can remove links and only mention `i18nify-react`. ignore the js since that will go away with the approach we discussed last time

Reply:
Done 
<img width="1045" alt="Screenshot 2024-02-12 at 3 05 56 PM" src="https://github.com/razorpay/blade/assets/21965506/e5eb99f8-f4e4-4655-8a87-c3be80669c5e">

Reply:
Done
<img width="1042" alt="Screenshot 2024-02-12 at 3 31 47 PM" src="https://github.com/razorpay/blade/assets/21965506/c94bbd5a-8f99-4ccc-bf68-677173693501">